rhp/v4/server.go

```diff
@@ -1101,13 +1101,19 @@ func (s *Server) handleRPCVerifySector(stream net.Conn) error {
 	}
 	prices, token := req.Prices, req.Token
 
+	if exists, err := s.sectors.HasSector(req.Root); err != nil {
```

Comment:
@n8maninger is it safe to debit the account after checking if the sector exists? Seems like it opens up the host to renters repeatedly forcing it to perform disk I/O without payment. Same goes for `RPCReadSector`.